src/SFML/Window/Win32/WglContext.cpp

```diff
@@ -137,17 +149,23 @@ m_ownsWindow   (false)
     // Save the creation settings
     m_settings = settings;
 
-    // Make sure that extensions are initialized if this is not the shared context
-    // The shared context is the context used to initialize the extensions
-    if (shared && shared->m_deviceContext)
-        ensureExtensionsInit(shared->m_deviceContext);
-
     // Create the rendering surface (window or pbuffer if supported)
     createSurface(shared, width, height, VideoMode::getDesktopMode().bitsPerPixel);
 
     // Create the context
     if (m_deviceContext)
```

Comment:
If possible, I would also DRY the ctors.

Reply:
Delegating constructors are coming in C++11. 😉

Reply:
Good! Maybe add a `\todo` so we actually clean up the code at some point?